### ISSUE TITLE
[Feature request] Keyboard keys

### ISSUE THREAD

**Issue by thomasdalbert** [axle-h/Retro.Net#9]
Let users play using keyboard keys.

**Comment by axle-h** [comment]
Good call. Hadn't thought of that. Should be pretty easy to implement too.

**Comment by RgeditV1** [comment]
> Good call. Hadn't thought of that. Should be pretty easy to implement too.

hi bro, ill try to implement it
